src/components/Explore/ExploreChart.tsx

```diff
@@ -42,12 +45,18 @@ const ExploreTooltip: React.FC<{
 
   return pointSmooth && pointRaw ? (
     <Tooltip
+      width={'150px'}
       top={top(pointSmooth)}
       left={left(pointSmooth)}
       title={moment(date).format('MMM D, YYYY')}
     >
       <Styles.TooltipSubtitle>{seriesRaw.tooltipLabel}</Styles.TooltipSubtitle>
-      <Styles.TooltipMetric>{formatInteger(pointRaw.y)}</Styles.TooltipMetric>
+      <Styles.TooltipMetric>
+        {formatInteger(pointSmooth.y)}
+      </Styles.TooltipMetric>
+      <Styles.TooltipSubtitle>
+        {`${formatInteger(pointRaw.y)} cases`}
```

Comment:
We seem to say "cases" even if it's deaths / hospitalizations / whatever...

![image](https://user-images.githubusercontent.com/206364/91789275-2deedb80-ebc3-11ea-8a8c-848cb5a682e5.png)

Reply:
FWIW- Feel free to ignore, but if it were me I'd maybe make the tooltip say:

> HOSPITALIZATIONS 
> 46
> **7-DAY AVG. 55**

which means for cases it'd be:

> CASES
> 249
> **7-DAY AVG. 206**

Basically we only need to say "cases" once.  And it makes sense to put the raw number before the averaged number, since the average number is derived from the raw number.  But we can still emphasize the averaged number by making it bold still.

🤷

Reply:
oops, yeah, that's a bug